src/pages/SettingsPage.js

```diff
@@ -94,6 +120,29 @@ const SettingsPage = ({
                             Sign Out
                         </Text>
                     </TouchableOpacity>
+                    <View style={[styles.settingsPageBody, styles.mt6]}>
+                        <Text style={[styles.settingsPageHeading, styles.mt2, styles.mb4]} numberOfLines={1}>
+                            Chats
+                        </Text>
+                        <Text style={[styles.textP]}>
+                            How should we display chats on your home screen?
+                        </Text>
+                        <View style={[styles.mt2, styles.mb2]}>
+                            <RNPickerSelect
+                                onValueChange={updatePriorityMode}
+                                items={Object.values(priorityModes)}
+                                style={{
+                                    ...styles.picker,
+                                }}
+                                useNativeAndroidPickerStyle={false}
+                                placeholder={{}}
```

Comment:
Is there a reason this is being passed an empty object?

Reply:
Yes. It is to make sure there is no placeholder. There is some default placeholder that is present if you don't pass the empty object.

Reply:
Hmm, but it always has a `value` so why does it need an explicitly empty placeholder?

Reply:
If there is some reason it would be good to note in a comment so we can avoid someone accidentally removing this (if not then we should be able to remove this).

Reply:
Ah I see the problem. The placeholder is a _selectable option_ in the list and not an actual placeholder. Yeah, would be good to add a comment for that. It's not super obvious that would happen.

Reply:
I've added a comment. It's outside the React component tag because the linter doesn't like empty lines inside but insists on empty lines above comments. this seemed the cleanest method.